### ISSUE TITLE
`attr-no-unsafe-char` line feed

### ISSUE THREAD

**Issue by alfaproject** [htmllint/htmllint#186]
Why is the line feed marked as an unsafe attribute character?

Pretty sure this perfectly valid HTML:

```html
<p style="
background: red;
color: white;
"></p>
```

The example is for science by the way.

**Comment by mlochbaum** [comment]
It's definitely valid, and I'm pretty sure there's no other reason to complain about it (as there would be with a more unusual character, which should be encoded with an escape). I can't find any justification at all for the particular character ranges used by `attr-no-unsafe-char`. I'll definitely remove LF and CR from those, and possibly some other characters.

**Comment by alfaproject** [comment]
Thank you @mlochbaum (:
